sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/fragment/organisms/InformationMessageCardViewFragment.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.hmrc.sample_compose_fragments.presentation.fragment.organisms
+
+import android.os.Bundle
+import android.view.View
+import android.widget.Toast
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import androidx.fragment.app.Fragment
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme
+import uk.gov.hmrc.components.compose.ui.theme.rememberWindowSizeClass
+import uk.gov.hmrc.sample_compose_components.R
+import uk.gov.hmrc.sample_compose_components.databinding.FragmentComposeExampleBinding
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.organisms.InformationMessageCardViewScreen
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.organisms.PrimaryCardViewScreen
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.sampletemplate.HmrcSurface
+
+class InformationMessageCardViewFragment : Fragment(R.layout.fragment_compose_example) {
+
+    private lateinit var binding: FragmentComposeExampleBinding
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        binding = FragmentComposeExampleBinding.bind(view)
+        binding.composeView.apply {
+            setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+            setContent {
+                val window = rememberWindowSizeClass()
+                HmrcTheme(window) {
+                    HmrcSurface {
+                        InformationMessageCardViewScreen { showToast(getString(R.string.cta_pressed)) }
```

Comment:
As click action has no usage on `InformationMessageCardViewScreen` that's `showToast(getString(R.string.cta_pressed))` is not invoked anymore. Do we need this?

Reply:
Removed